oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsUsbDevice.java

```diff
@@ -55,269 +57,256 @@
  */
 public class WindowsUsbDevice extends AbstractUsbDevice {
 
-    private static final Logger LOG = LoggerFactory.getLogger(WindowsUsbDevice.class);
+	private static final Logger LOG = LoggerFactory.getLogger(WindowsUsbDevice.class);
 
-    enum USBControllerProperty {
-        PNPDEVICEID;
-    }
+	enum USBControllerProperty {
+		PNPDEVICEID;
+	}
 
-    // List of controllers
-    private static List<String> controllerDeviceIds = null;
+	// List of controllers
+	private static Supplier<List<String>> controllerDeviceIds = Memoizer.memoize(WindowsUsbDevice::getControllerDeviceIdList);
```

Comment:
One more small request.  Elsewhere in OSHI I've done a static import of the `Memoizer.memoize` method to shorten this to just `memoize(...)`.  Would be nice to do that here for consistency.

Reply:
I'll go fix that right now. I just reformatted the code to the OSHI standard, but every time I try to commit the new code, it says "Changes not staged for commit:" Can you help me with this issue?

Reply:
You need to commit the changed files!  Command line is `git commit` but there should be a menu option in Eclipse under the "Team" menu.

Reply:
Also, once I remove the "Memoizer.", the code no longer compiles. The error says "The method memoize(WindowsUsbDevice::getControllerDeviceIdList) is undefined for the type WindowsUsbDevice" Will this be an issue?

Reply:
You have to add an `import static oshi.util.Memoizer.memoize;` statement.

Reply:
Alright, I believe the changes have been committed and pushed. Can you verify this for me please?